WordPress/src/main/res/values/strings.xml

```diff
@@ -5130,6 +5130,8 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="ai_bot_empty_conversations_title">No conversations yet</string>
     <string name="ai_bot_empty_conversations_message">Start a new conversation to get help with your WordPress site or account.</string>
     <string name="ai_bot_empty_conversations_button">Start conversation</string>
+    <string name="ai_bot_you">You</string>
+    <string name="ai_bot_odie" translatable="false">Odie</string>
```

Comment:
Do we surface the name "Odie" anywhere else? If not, this might be confusing. Should we use a generic "Bot" instead?